src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -348,20 +347,23 @@ private JObject GetVerificationResult(UInt160 scriptHash, ContractParameter[] ar
             {
                 throw new RpcException(-100, "Unknown contract");
             }
-            var methodName = "verify";
+            var md = contract.Manifest.Abi.GetMethod("verify", -1);
+            if (md is null)
+                throw new RpcException(-101, $"The smart contract {contract.Hash} haven't got verify method without arguments");
+            if (md.ReturnType != ContractParameterType.Boolean)
+                throw new RpcException(-102, "The verify method doesn't return boolean value.");
 
             Transaction tx = signers == null ? null : new Transaction
             {
                 Signers = signers.GetSigners(),
-                Attributes = Array.Empty<TransactionAttribute>()
+                Attributes = Array.Empty<TransactionAttribute>(),
```

Comment:
Can we also fill the transaction script? I think that []byte{OpCode.RET} is enough. 

This is needed for those `verify` methods that use `System.Runtime.GetScriptContainer` interop, because transaction can't be serialized without a script.

Reply:
Updated.